src/ui/public/persisted_state/__tests__/persisted_state_provider.js

```diff
@@ -32,7 +32,7 @@ describe('Persisted State', function () {
 
     it('should be an event emitter', function () {
       persistedState = new PersistedState();
-      expect(persistedState).to.be.an(Events);
+      expect(persistedState._emitter).to.be(Events);
```

Comment:
No need to test the internals of the `PersistedState` class. This test should either be updated to ensure that the object includes the expected methods or removed.